modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/util/BinaryExpressionConverterUtil.java

```diff
@@ -35,6 +39,10 @@ else if (Objects.equals(BinaryExpression.Operation.LT, operation)) {
 			return column.lt(value);
 		}
 		else if (Objects.equals(BinaryExpression.Operation.NE, operation)) {
+			if (value == null) {
```

Comment:
could you change to `Validator.isNull(value)` ?

Reply:
We can't use that here because we're really only interested in the actual null reference and `Validator.isNull(value)` will return true in other [scenarios](https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/Validator.java#L911)